charmcraft.yaml

```diff
@@ -31,13 +31,24 @@ bases:
       channel: "22.04"
       architectures: ["arm64"]
 
+charm-libs:
+  - lib: "grafana_k8s.grafana_source"
+    version: "0"
+
 containers:
   kiali:
     resource: kiali-image
 
-provides:
+peers:
+  # TODO: Can we remove this?  Required for the grafana_datasource lib, but not for features we use.
+  grafana:
+    interface: grafana_peers
```

Comment:
Can we not call the peer relation "grafana"?

Reply:
Annoyingly, [no](https://github.com/canonical/grafana-k8s-operator/blob/807317c9f0943287c3650d5ef0f072fd06780387/lib/charms/grafana_k8s/v0/grafana_source.py#L539).  It isn't exposed as something configurable by the lib

We can change the lib easily enough and then yes.  Since we plan to replace this with a slimmer relation anyway, I'm like 51% against bothering to change the grafana lib.  But I'm open to changing it if you feel strongly

Reply:
Wow that's lame. No we shouldn't bother to change the lib.